### ISSUE TITLE
Weird intermittent static when playing audio using ALSA

### ISSUE THREAD

**Issue by Synergyst** [Audio-Injector/Octo#61]
I am not sure what the issue is as I've searched the mailing list and Github questions and only found one person with the same issue.
I could not find if they discovered a solution however.

Here is their video of the exact problem I have: 
https://www.youtube.com/watch?v=Cs7npai_eEU

Any help would be greatly appreciated! :)

**Comment by Synergyst** [comment]
I probably should have mentioned some extra things.

I am using a RPi3B with a 32-bit installation of Raspbian Lite.
Pulseaudio is removed.
It exhibits this issue no matter if I use ALSA or JACK.
```
pi@rpi3:~ $ uname -anr
Linux rpi3 6.1.21-v7+ #1639 SMP Fri Mar 24 16:56:29 GMT 2023 armv7l GNU/Linux
```

I have tried `dtoverlay=audioinjector-addons,non_stop_clocks` and `dtoverlay=audioinjector-addons` with the same results. 

Here is my truncated (comments removed) /boot/config.txt:
```
framebuffer_width=1920
framebuffer_height=1080
hdmi_force_hotplug=1
arm_freq=500
dtparam=i2c_arm=on
dtparam=i2s=on
camera_auto_detect=0
display_auto_detect=0
max_framebuffers=2
disable_overscan=1
dtoverlay=disable-bt
dtoverlay=disable-wifi
[cm4]
otg_mode=1
[all]
[pi4]
arm_boost=1
[all]
gpu_mem=384
dtoverlay=audioinjector-addons,non_stop_clocks
```

**Comment by Synergyst** [comment]
[octo-issue-audacity-examples.zip](https://github.com/Audio-Injector/Octo/files/11114713/octo-issue-audacity-examples.zip)
Here is a screenshot of the wave form as well as the actual audio which was recorded of the issue.

EDIT: I am observing the same behavior now also on my CM4 which I installed it onto instead of my RPi3B for testing.
I am using the standard official CM4 IO board for context if that matters.

**Comment by Synergyst** [comment]
I found a solution that someone posted about regarding another issue!
#49 

_Albeit it's somewhat of a nasty one, it is **a solution:**_ 
https://github.com/whofferbert/audioinjector-octo-setup#kernel-problems

TL;DR:
You must downgrade your kernel to 4.19.118.
There may be other versions which work, though this is pretty bad in my opinion. At least it works! :)
